doc/content/source/postprocessors/BetaEffective.md

```diff
@@ -0,0 +1,17 @@
+# BetaEffective
+
+## Description
+
+`BetaEffective` computes $\beta_{eff}$ given that the [OpenMC problem](OpenMCCellAverageProblem.md)
```

Comment:
For folks who click on this page perhaps without knowing what beta_eff and Lambda_eff are, it would be good to add a 1-2 sentence description of what these parameters mean/represent.